db/schema.rb

```diff
@@ -545,6 +545,18 @@
     t.index ["volume"], name: "index_memberships_on_volume"
   end
 
+  create_table "motions", id: :serial, force: :cascade do |t|
```

Comment:
can you take these (the montions and votes tables) out please? It's interesting that your db has them. They were removed years ago